copulas/__init__.py

```diff
@@ -311,7 +311,13 @@ def _get_addon_target(addon_path_name):
 def _find_addons():
     """Find and load all copulas add-ons."""
     group = 'copulas_modules'
-    for entry_point in iter_entry_points(group=group):
+    try:
+        eps = entry_points(group=group)
+    except TypeError:
+        # Load-time selection requires Python >= 3.10 or importlib_metadata >= 3.6.
+        eps = entry_points().get(group, [])
```

Comment:
does entry_points().select(group=group) work for both?

Reply:
I'm not sure `select` [existed](https://docs.python.org/3.8/library/importlib.metadata.html) in py38, but I can run GitHub Actions to double check.